scholia/app/templates/base.html

```diff
@@ -201,6 +201,24 @@
      });
  }
 
+ $('#searchterm').keyup(function( e ) {
+      var q = $('#searchterm').val();
+      $.getJSON("https://www.wikidata.org/w/api.php?callback=?", {
+          search: q,
+          action: "wbsearchentities",
+          language: "en",
+          uselang: "en",
+          format: "json",
+          strictlanguage: true,
+        },
+        function(data) {
+          $("#searchresult").empty();
+          $.each(data.search, function(i, item) {
+            $("#searchresult").append("<div><a href='" + item.title + "'>" + item.label + "</a> - " + item.description + "</div>");
```

Comment:
I think with need a "./" here.

Reply:
No that is not enough. There is a wrong redirect

Reply:
I think there might also be an escape issue

Reply:
Where in this line? it seems to work fine on my end

Reply:
I have just added the escape myself.